docs/openapi/components/schemas/common/AgricultureActivity.yml

```diff
@@ -47,7 +47,8 @@ properties:
     type: string
     $linkedData:
       term: activityDate
-      '@id': https://schema.org/DateTime
+      '@id': https://schema.org/endDate
+      '@type': http://www.w3.org/2001/XMLSchema#date     
```

Comment:
this RDF type is not correct i think see

https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L119

Reply:
@OR13 do you mean that this is `http://www.w3.org/2001/XMLSchema#date` & not `http://www.w3.org/2001/XMLSchema#dateTime
http://www.w3.org/2001/XMLSchema#date`? If so that's intentional, I think this one's not supposed to have a time attached.

Unless I'm misremembering the approach we settled on for the 2/6 trace meeting? Did we say that we want all instances of "date" to be "datetime" but with the implicit assumption that if there's no specified time it's midnight? We neglected to upload notes that meeting & my own are kinda spotty 😅

Reply:
#date is the correct granularity.

Reply:
And endDate is better than DateTime... Not good, but better

Reply:
`validUntil` is appropriately a datetime.

`activityDate` is appropriately a date.